app/Equipment.py

```diff
@@ -0,0 +1,47 @@
+from typing import Any
```

Comment:
The import statement `from typing import Any` is correct, but it is not used in the code. If you don't plan to use `Any`, consider removing this import to clean up the code.